content/docs/hooks-custom.md

```diff
@@ -184,24 +183,24 @@ function ChatRecipientPicker() {
   const isRecipientOnline = useFriendStatus(recipientID);
 ```
 
-This lets us know whether the *currently selected* friend is online. If we pick a different friend and update the `recipientID` state variable, our `useFriendStatus` Hook will unsubscribe from the previously selected friend, and subscribe to the status of the newly selected one.
+Esto nos permite saber cuando el amigo* actualmente seleccionado* esta en linea. Si elegimos un amigo diferente y actualizamos la variable de estado `recipientID`, nuestro Hook `useFriendStatus` va a desuscribirse de el amigo previamente seleccionado, y suscribirse al estado de el nuevo seleccionado.
 
-## `useYourImagination()`
+## `usaTuImaginación()`
 
-Custom Hooks offer the flexibility of sharing logic that wasn't possible in React components before. You can write custom Hooks that cover a wide range of use cases like form handling, animation, declarative subscriptions, timers, and probably many more we haven't considered. What's more, you can build Hooks that are just as easy to use as React's built-in features.
+Los Hooks personalizados ofrecen la flexibilidad de compartir la lógica que no era posible con los componentes de React antes. Puedes escribir Hooks personalizados que cubran una amplia gama de casos de uso como manejo de formularios, animación, suscripciones declarativas, temporizadores y probablemente muchos mas que no hemos considerado. Ademas, puedes construir Hooks que sean tan fáciles de usar como las características incorporadas de React.
 
-Try to resist adding abstraction too early. Now that function components can do more, it's likely that the average function component in your codebase will become longer. This is normal -- don't feel like you *have to* immediately split it into Hooks. But we also encourage you to start spotting cases where a custom Hook could hide complex logic behind a simple interface, or help untangle a messy component.
+Intenta resistirte a añadir la abstracción demasiado pronto. Ahora que los componentes funcionales pueden hacer mas, es probable que el promedio de componentes funcionales en tu base de código se amplíe. Esto es normal -- no te sientas como si *tuvieras* que dividirlo inmediatamente en Hooks. Pero también te animamos a empezar a descubrir casos donde un Hook personalizado podría ocultar la lógica compleja detrás de una interfaz simple, o ayudar a desenredar un componente desordenado.
 
-For example, maybe you have a complex component that contains a lot of local state that is managed in an ad-hoc way. `useState` doesn't make centralizing the update logic any easier so might you prefer to write it as a [Redux](https://redux.js.org/) reducer:
+Por ejemplo, quizás tienes un componente complejo que contiene muchos estados locales que son gestionados en una forma ad-hoc. El `useState` no hace la centralización de actualización de lógica mas fácil, así que podrías preferir escribirlo como un reductor [Redux](https://redux.js.org/):
```

Comment:
```suggestion
Por ejemplo, quizás tienes un componente complejo que contiene muchos estados locales que son gestionados en una forma *ad-hoc*. El `useState` no hace la centralización de actualización de lógica mas fácil, así que podrías preferir escribirlo como un reductor [Redux](https://redux.js.org/):
```

Reply:
You used `muchos estados locales`, that means `lots of local states`, in plural. But they say `a lot of local state`, which would become `mucho estado local` or `una gran cantidad de estado local`.

If we consider `useState` a name, then the article `El` on the second sentence shouldn't be necessary.